jscomp/main/js_main.ml

```diff
@@ -289,7 +287,8 @@ let buckle_script_flags : (string * Bsc_args.spec * string) array =
     (******************************************************************************)
 
 
-    "-bs-super-errors", unit_lazy Super_main.setup,
+    (* XXX(anmonteiro): flag kept for compatibility. *)
```

Comment:
Never seen this comment syntax before, what does `XXX` mean?

Reply:
Leaving this SO answer here for posterity: https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

I use it in the sense of "heads up, this needs attention".

Reply:
Ahaa, neat! Do you sign your name between the parentheses as the person who added it or is it “look here, future @anmonteiro)?

Reply:
it's the person who added, so I can know who to bother in the future. In this case, my past self :)

Reply:
Gotcha
![Thumbs Up but weeb](https://media.discordapp.net/attachments/477991887631024138/823347431356760135/Alfa_thumbsup_FB.gif)
Feel free to mark as resolved or keep it open to be easier to notice by others ❤️